bin/node-gyp.js

```diff
@@ -12,6 +12,7 @@ process.title = 'node-gyp'
 
 var gyp = require('../')
 var log = require('npmlog')
+var path = require("path");
```

Comment:
I'll fix the style problem, I care more about if the environment solution is acceptable